app/models/post.js

```diff
@@ -63,6 +64,19 @@ export default Model.extend(ValidationEngine, {
     scratch: null,
     titleScratch: null,
 
+    // Computed Date properties
+    // timeZone.offset service returns the moment-timezone from the
+    // activeTimezone, set in the blogs settings
+
+    publishedAtOffset: computed('timeZone.offset', 'publishedAt', function() {
```

Comment:
I'm wondering if `publishedAtOffset` is actually needed at the model layer - right now it's only being used to pass its value into `gh-datetime-input` component which then adjusts it to match the blog timezone anyway so we could simply pass the UTC `publishedAt` value to the component instead.

I think removing it from the model layer will help to keep any offset adjustments outside of the main code and in the display layer. The `*At`->`*AtUTC` rename would also become cleaner as it will only be the core model attributes changing name to make it clear that we always deal with UTC internally.